tests/src/test/scala/whisk/core/containerpool/test/ContainerArgsConfigTest.scala

```diff
@@ -52,7 +52,7 @@ class ContainerArgsConfigTest extends FlatSpec with Matchers {
     config.dnsServers shouldBe Seq[String]("google.com", "1.2.3.4")
     //check map parsing of extra-args config
     config.extraArgs.get("label") shouldBe Some(Set("l1", "l2", "l3"))
-    config.extraArgs.get("environment") shouldBe Some(Set("e1", "e2"))
+    config.extraArgs.get("env") shouldBe Some(Set("e1", "e2"))
```

Comment:
i suggest a test for extra args following the comment above, or fix the comment.

Reply:
I think this is addressed now